Composer/packages/client/src/components/ProjectTree/TriggerCreationModal.tsx

```diff
@@ -413,7 +413,7 @@ export const TriggerCreationModal: React.FC<TriggerCreationModalProps> = (props)
             <TextField
               data-testid="RegExField"
               errorMessage={formData.errors.regEx}
-              label={formatMessage('Please input regex pattern')}
+              label={formatMessage('Please input regEx pattern')}
```

Comment:
Should this say 'regular expression pattern' and avoid the abbreviation?

Reply:
Good question. This should be part of the same discussion as the "Ok/OK/Okay" thing.